test/e2e/storage/drivers/csi.go

```diff
@@ -614,17 +641,17 @@ func (g *gcePDCSIDriver) PrepareTest(f *framework.Framework) (*storageframework.
 	// Cleanup CSI driver and namespaces. This function needs to be idempotent and can be
 	// concurrently called from defer (or AfterEach) and AfterSuite action hooks.
 	cleanupFunc := func() {
-		ginkgo.By(fmt.Sprintf("deleting the test namespace: %s", ns1))
-		// Delete the primary namespace but its okay to fail here because this namespace will
+		ginkgo.By(fmt.Sprintf("deleting the test namespace: %s", testns))
```

Comment:
Unrelated side note, it seems like all drivers in this file have basically copy pasted the cleanup funcs. We could probably clean this up with a common function? Can you open a cleanup issue and we can find someone to work on it?

Reply:
I can do it, doesn't seem too much work :)

Reply:
The mock driver does an extra step of deleting a configmap. Would creating a single function and adding a way to to hook in this logic be too much complexity?

Reply:
Would the configmap cleanup be covered by the namespace cleanup?

Reply:
It should. This logic also existed before the namespace cleanup was added, so it might've been a mistake in keeping it.

I'll do it as a separate PR tho in case something breaks

Reply:
Created a tracking bug here just in case, but I'm hoping to get to it soon after this PR merges: https://github.com/kubernetes/kubernetes/issues/99054